### ISSUE TITLE
Dashboard no data when suite.xml contains suite-file

### ISSUE THREAD

**Issue by social2wifi** [qmetry/qaf#265]
### QAF Version
> Note: only the latest version is supported

### Steps To Reproduce
    1. Create suite.xml
    2. In this file contains something like
```xml
<suite name="Master Suite">
	<suite-files>
		<suite-file path="suite1.xml"></suite-file>
		<suite-file path="suite2.xml"></suite-file>
	</suite-files>
</suite>
```
    3. When execution is finished then in dashboard we can view a Master Suite execution but no info in Overview.

**Comment by cjayswal** [comment]
This relates to #23. At present, as per design, JSON reporting supported only for one suite.

**Comment by cjayswal** [comment]
#23 available with 2.1.15 beta.
